esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -6520,11 +6574,23 @@ bool CWsDfuEx::onDFUFilePublish(IEspContext &context, IEspDFUFilePublishRequest
                  throw makeStringException(ECLWATCH_INVALID_INPUT, "DFUFilePublish: No ECLRecordDefinition defined.");
 
 #ifdef _CONTAINERIZED
-            IERRLOG("CONTAINERIZED(CWsDfuEx::onDFUFilePublish)");
-            ClusterType clusterType = NoCluster;
+            const char *planeName = fileIdItems.item(1);
+            groupName = planeName;
```

Comment:
With groupname being set no matter what, is there a need for the separate planeName variable?

Reply:
The format/content of this string is created by onDFUFileCreateV2, and opaque to the client (simply passed back).
In container mode this field will always be the plane name.
It is really keeping/setting 'groupName' here purely so the logging on line 6677 doesn't have conditional code where 1 uses planeName and other uses groupName.